src/index.html

```diff
@@ -5,25 +5,73 @@
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>API-based App</title>
-  <link rel="stylesheet" href="../src/styles.css" />
+  <title>La Collezione</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Italianno&family=Nunito:ital,wght@0,400;0,700;1,400&family=Raleway&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" />
+  <link rel="stylesheet" href="./styles.css" />
 </head>
 
-<body class="mx-auto h-full w-full bg-pink-500">
-  <h1 class="
-        p-24
-        text-center
-        justify-center
-        items-center
-        text-4xl
-        font-bold font-mono
-        text-white
-      ">
-    This template is pre-configured with Tailwind CSS and contains the basic
-    structure for a project requiring vanilla JS.
-  </h1>
-  <script src="index.js"></script>
+<body class="mx-auto h-full w-full">
+  <!-- Nav -->
+  <div class="main-container">
+    <nav id="nav" class="nav-container">
+      <figure class="flex items-center align-middle">
+        <img class="w-8 lg:w-12 text-center" src="/src/art-museum.png" alt="logo">
+        <figcaption id="logo">&nbsp; La Collezione
+        </figcaption>
+      </figure>
+      <div class="nav-items">
+        <div id="collection">The Collection</div>
+        <div id="artworks">Exhibitions</div>
+        <div id="art">Art</div>
+        <div id="reserve">Reserve</div>
+      </div>
+    </nav>
```

Comment:
- [ ] The header/navigation should be similar in design to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).
![Screenshot from 2021-10-18 03-41-40](https://user-images.githubusercontent.com/65501535/137651265-f79c45af-838d-482c-9b2f-10c2ff060bcb.png)